Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.8-slim
+
+ARG VERSION
```

Comment:
Is there a default you can set this to, so that if it's not given it'll just use the latest?

Reply:
The way to do that is `ARG=<default value>`, but I need to find a value that'll tell pip to install the latest available version.

Reply:
Could maybe do this?:

```
ARG VERSION=""

RUN [ -z "$VERSION" ] && pip3 install bork || pip3 install "bork==${VERSION}"
```

but idk how i feel about that.

Reply:
A hacky "works for now i guess" type of approach would be to just hard-code the latest released version in the Dockerfile and bump it with setup.cfg.

Reply:
Reasons I wrote buildahscript